app/assets/javascripts/initializers/initializeOnboardingTaskCard.js.erb

```diff
@@ -0,0 +1,9 @@
+function initializeOnboardingTaskCard() {
+  if (localStorage.getItem('task-card-closed') === 'yes') { return }
+
+  var taskCard = document.getElementsByClassName("onboarding-task-card")[0];
+
+  if (taskCard !== undefined) {
```

Comment:
```suggestion
  if (taskCard) {
```
If we find a `taskCard` by class name in the previous line, then is this should work, right? IIRC, `undefined` is falsy in JS, so even `null` or an empty string would pass this check? 🤔

Reply:
You are right that would work! I got bitten by a null safety before which led me to do this janky alternative.